src/ClassicUO.Assets/AnimationsLoader.cs

```diff
@@ -130,76 +130,73 @@ private unsafe void LoadInternal()
+                        );
 
-                            if (parts.Length < 3)
-                            {
-                                continue;
-                            }
+                        if (parts.Length < 3)
+                        {
+                            continue;
+                        }
 
-                            int id = int.Parse(parts[0]);
-                            string testType = parts[1].ToLower();
-                            int commentIdx = parts[2].IndexOf('#');
+                        int id = int.Parse(parts[0]);
+                        string testType = parts[1].ToLower();
+                        int commentIdx = parts[2].IndexOf('#');
 
-                            if (commentIdx > 0)
-                            {
-                                parts[2] = parts[2].Substring(0, commentIdx - 1);
-                            }
-                            else if (commentIdx == 0)
-                            {
-                                continue;
-                            }
+                        if (commentIdx > 0)
+                        {
+                            parts[2] = parts[2].Substring(0, commentIdx - 1);
+                        }
+                        else if (commentIdx == 0)
+                        {
+                            continue;
+                        }
 
-                            uint number = uint.Parse(parts[2], NumberStyles.HexNumber);
+                        uint number = uint.Parse(parts[2], NumberStyles.HexNumber);
 
-                            for (int i = 0; i < 5; i++)
-                            {
-                                if (
-                                    testType.Equals(
-                                        typeNames[i],
-                                        StringComparison.InvariantCultureIgnoreCase
-                                    )
+                        for (int i = 0; i < 5; i++)
+                        {
+                            if (
+                                testType.Equals(
+                                    typeNames[i],
+                                    StringComparison.InvariantCultureIgnoreCase
                                 )
+                            )
+                            {
+                                _mobTypes[id] = new MobTypeInfo()
                                 {
-                                    _mobTypes[id] = new MobTypeInfo()
-                                    {
-                                        Type = (AnimationGroupsType)i,
-                                        Flags = (AnimationFlags )(0x80000000 | number)
-                                    };
-
-                                    break;
-                                }
+                                    Type = (AnimationGroupsType)i,
+                                    Flags = (AnimationFlags )(0x80000000 | number)
```

Comment:
Might as well fix the bad syntax styling I guess.

```suggestion
                                    Flags = (AnimationFlags)(0x80000000 | number)
```

Reply:
Yes, but this has nothing to do with this PR. The line is only touched because one indentation level was removed. Changing indentation *and* doing code changes is what I avoid, because it effectively hides changes from review.

Reply:
Ok that is fine